### PR TITLE
Update Contributing.md

### PR DESCRIPTION
**Pull request by Joshix-1**
Require successful checks for merging and make conditions a little bit stricter.
No PR with only one upvote and failing checks should be merged after 74h

## Type of change
- Updated merge conditions

# Checklist
- The project compiles and runs correctly

### CODE REVIEW

**Comment by ASecondGuy** [comment]
I don't like the wording. Maybe something like this?

All must be true:
- No checks are failing
- upvotes - downvotes > 2
- 75% positive votes
- One must be true: 
  - **Last commit is 24h or older** and **has more than 10 votes** 
  - **Last commit is 72h or older**

**Comment by Joshix-1** [comment]
I disagree, I think that is worse

**Comment by Numenter** [comment]
>"No checks are failing"

We should implement the Branch protection rules https://github.com/letsgamedev/Suffragium/issues/44 for that. ( Require status checks to pass before merging )
Didn't had the motivation to create a PR for it. Maybe someone else wants to do it.

>"upvotes - downvotes > 2"

Looks like a 'min vote of 3' to me.

**Comment by Joshix-1** [comment]
> We should implement the Branch protection rules https://github.com/letsgamedev/Suffragium/issues/44 for that.

Agree, but writing it down is still a good idea. As that's then for everyone visible a rule.

> Looks like a 'min vote of 3' to me.

Yes, but better (I think). With a downvote you then need at least 4 upvotes.

**Comment by Numenter** [comment]
I hate to say it. Suffragium is in a constant decline. Stricter rules don't help and they could hinder any progress in the future.

> Yes, but better (I think). With a downvote you then need at least 4 upvotes.

And now you need 3 upvotes (3up 1down). I thought 75 hours was enough time to vote.
If you don't vote, you don't count. If a PR goes through because nobody participated, who cares. Clearly nobody.

I'm withholding my vote for this PR, but it's clear how I stand.

**Comment by RedstoneMedia** [comment]
I agree. A lot of PRs are struggling to even get one or two thumbs up, just because there is so little activity. Making the rules stricter now **will** accelerate the decline of this project, because there are even longer wait times and thus less activity.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,10 +35,14 @@ Place a :+1: or :-1: emoji on the initial post of an pull request. When the merg
...
-- **Last commit is 24h or older** and **has more than 10 votes** and **75% positive votes**
-- **Last commit is 72h or older** and **75% positive votes**
+Necessary conditions:
+- No checks are failing
+- at least one upvote, that isn't from the creator of the pr
+- 75% positive votes
+
+If **additionally** one of the following conditions is true, the PR will me merged.
+- **Last commit is 24h or older** and **has more than 10 votes**
+- **Last commit is 72h or older**
```
Comment (Numenter) [inline CONTRIBUTING.md:45]:
```suggestion
When **all conditions** are met, the PR will be merged.
- All checks have passed
- 75% positive votes (Creator vote does not count)
- Last commit is older than 24h and has more than 10 votes **or** Last commit is older than 72h
```

**Review by Numenter — CHANGES_REQUESTED** [review]
I would simplify the conditions.

**Comment by RedstoneMedia** [comment]
I would remove the requirement, where the creator can not vote on his own PR. This forces other users to look at the new PRs every couple of days, if they don't like it they just give it a dislike. This extends the lifetime of the project and could even rejuvenate it, once everyone of the original voters looses interest and there is just one person left.

**Comment by Joshix-1** [comment]
I think a PR should need at least one upvote of another person.

**Pull request closed without merging**